compiler/src/dotty/tools/dotc/typer/Applications.scala

```diff
@@ -2042,7 +2052,7 @@ trait Applications extends Compatibility {
       skip(alt.widen)
 
     def resultIsMethod(tp: Type): Boolean = tp.widen.stripPoly match
```

Comment:
```suggestion
    def resultIsMethod(tp: Type): Boolean = tp.widen match
```

Reply:
Update:
The above suggestion is wrong 
As the `.stripPoly` is there to ensure we "apply" the method to one set or arguments, and look at what remains